deltacat/utils/ray_utils/retry_handler/retry_strategy_config.py

```diff
@@ -0,0 +1,14 @@
+from typing import List, Optional
+
+from ray_manager.models.ray_remote_task_exception_retry_strategy_config import RayRemoteTaskExceptionRetryConfig
+
+def get_retry_strategy_config_for_known_exception(exception: Exception, exception_retry_strategy_configs: List[RayRemoteTaskExceptionRetryConfig]) -> Optional[RayRemoteTaskExceptionRetryConfig]:
```

Comment:
The AIMD retry strategy can be baked into this. Why do we have retry strategy specified at two different places?